packages/create-sitecore-jss/src/templates/nextjs-sxa/src/Layout.tsx

```diff
@@ -31,24 +32,45 @@ const Layout = ({ layoutData, headLinks }: LayoutProps): JSX.Element => {
     <>
       <Scripts />
       <Head>
-        <title>{fields?.Title?.value?.toString() || 'Page'}</title>
-        <link rel="icon" href={`${publicUrl}/favicon.ico`} />
+        <title>
+        <% if (xmcloud) {-%>
+          {layoutData.sitecore.context.renderingType === 'component'
```

Comment:
Is there any need for adjusting a Page title? It will be used only inside iframe, so it won't be visible. On the other hand it complicates Layout with additional conditions